### DISCUSSION TITLE
Import different JSON format

### DISCUSSION THREAD

**Discussion by register4gk** [directus/directus#19729] [Feature Requests]
### Summary

Dear all
I would like to bulk upload a large json file (190mb) similar to
https://learndirectus.com/how-to-import-data-into-directus-using-json/

However the structure is as follows:
[
"document_id": {"title": ..., "link": ..., "eurovoc_classifiers": ..., "full_text": ...}
"document_id": {"title": ..., "link": ..., "eurovoc_classifiers": ..., "full_text": ...}
]

### Motivation

I would like to upload large json files extracted by a scraper ... I can not change the scraper

### Detailed Design

Change the JSON importer to handle different structures

### Must Have:
- Json upload handles different structures

### Could Have:
- Better HowTo Page if the feature already exists

### Drawbacks

N/A

### Alternatives

- Thought about python script transforming the JSON structure, however this seems to be complicated

### Adoption Strategy

Dont know

**Comment by br41nslug** [comment]
I don't think changing the import json structure for this one specific usecase makes sense for the Directus platform core 😬 This can be done as extension or using a one-off json transformer (as suggested in alternatives)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
